README.md

```diff
@@ -1,12 +1,14 @@
 # Cloudbeat 
 ![Coverage](https://img.shields.io/badge/Coverage-46.4%25-yellow)
 [![Go Report Card](https://goreportcard.com/badge/github.com/elastic/cloudbeat)](https://goreportcard.com/report/github.com/elastic/cloudbeat)
-
+[![Build Status](https://internal-ci.elastic.co/job/cloudbeat/job/cloudbeat-mbp/job/main/badge/icon)](https://internal-ci.elastic.co/job/cloudbeat/job/cloudbeat-mbp/job/main/)
 ## Table of contents
-- [Prerequisites](#prerequisites)
-- [Running Cloudbeat](#running-cloudbeat)
-  - [Clean up](#clean-up)
-  - [Remote Debugging](#remote-debugging)
+- [Cloudbeat](#cloudbeat)
+  - [Table of contents](#table-of-contents)
+  - [Prerequisites](#prerequisites)
+  - [Running Cloudbeat](#running-cloudbeat)
+    - [Clean up](#clean-up)
+    - [Remote Debugging](#remote-debugging)
```

Comment:
I think we can remove these additions to the table of contents as they seem redundant, is it auto-generated somehow?

Reply:
Nope, i just added the badgeline